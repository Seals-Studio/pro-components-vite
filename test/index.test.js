import { mount } from '@vue/test-utils'
import BlockLayout from '../src/BlockLayout'

describe('BlockLayout', () => {

  it('should render BlockLayout', () => {
    const wrapper = mount({
      render () {
        return (
          <div>
           <BlockLayout>
             <span>demo</span>
           </BlockLayout>
          </div>
        )
      }
    })
    expect(wrapper).toMatchSnapshot()
  })

})
