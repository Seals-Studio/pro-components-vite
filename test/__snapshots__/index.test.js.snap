// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockLayout should render BlockLayout 1`] = `<div><span>demo</span></div>`;
